src/js/components/sharedComponents/header/InfoBanner.jsx

```diff
@@ -20,14 +22,29 @@ export default class InfoBanner extends React.Component {
     }
 
     render() {
-        const title = 'New to USAspending: COVID-19 Spending Data';
-        const content = (
+        let title = 'New to USAspending: COVID-19 Spending Data';
+        let content = (
             <p>
             USAspending now has spending data from federal agencies related to the Coronavirus Aid, Relief, and Economic Security (CARES) Act and other COVID-19 appropriations.
                 <button onClick={this.props.triggerModal}> Learn more</button> about the new data and features, or <Link to="/disaster/covid-19">visit the profile page</Link> to explore and download the data today!
             </p>
         );
 
+        if (isBefore(startOfToday(), new Date(2022, 1, 17))) {
```

Comment:
This says 1/17, but the Header checks 1/18. Will that cause problems or a missing banner?

Reply:
Nice catch. Thank you. This has been updated to match the date in Header.